unlock-app/src/components/interface/checkout/main/Payment.tsx

```diff
@@ -59,12 +59,12 @@ export function Payment({ injectedProvider, checkoutService }: Props) {
   const { isLoading, data: fiatPricing } = useQuery(
     ['fiat', lock.network, lock.address, recipients.length],
     async () => {
-      const pricing = await getFiatPricing(
+      const pricing = await getFiatPricing({
```

Comment:
We should removing the use of this API and instead getting the price directly for lock. 🤔